src/utils/identifiers.ts

```diff
@@ -33,4 +34,21 @@ export const solanaId = (address: string): string => {
 export const solanaTrackId = (address: string, id: string): string => {
   const suffix = id !== '' ? `/${id}` : '';
   return solanaId(address) + suffix;
-}
\ No newline at end of file
+}
+
+export const controlledEthereumAddressFromId = (id: string | undefined): string | undefined => {
```

Comment:
i'd just call this ethereumAddressFromId since there's nothing control-specific about it

Reply:
also this should be a string, never undefined